src/main/java/com/labsynch/labseer/service/AuthorServiceImpl.java

```diff
@@ -334,6 +334,7 @@ public Author signupAuthor(Author author) {
 		SimpleMailMessage mail = new SimpleMailMessage();
 		mail.setTo(author.getEmailAddress());
 		mail.setSubject("User Activation");
+		if(propertiesUtilService.getEmailFromAddress() != null && !propertiesUtilService.getEmailFromAddress().equals("") ) mail.setFrom(propertiesUtilService.getEmailFromAddress());
```

Comment:
If mail.setFrom() is empty, what happens?

Reply:
Okay, my guess is exception is thrown on Line 345 and we just just get a notice in the log file?  How do ACAS users without back end access find out there's a problem?

Reply:
java mail defaults to the current user/hostname.  So typically this is ok if you are using gmail because gmail doesn't care about the from address. It uses whoever the authentication user is.  With Amazon SES, the from user is what shows up in the email and the autentication user is just the IAM credentials for the SES user.

Reply:
Also, if an email fails to send, it shows up in the logs.